File: patterns/c-patterns/pattern25.c
// Pattern 25: ...

// ****A
// ***BA
// **CBA
// *DCBA
// EDCBA


#include <stdio.h>
int main()
{
    int i, j;
    int n=4;
    for(i=0; i<=n; i++)
    {
        for(j=n; j>=0; j--)
        {
            if(j>i)
            printf("*");
            else
            printf("%c", j+65);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern25.js
// Pattern 25: ...

// ****A
// ***BA
// **CBA
// *DCBA
// EDCBA

const n = 4;

for (let i = 0; i <= n; i++) {
    let row = '';

    for (let j = n; j >= 0; j--) {
        if (j > i) {
            row += '*';
        } else {
            row += String.fromCharCode(j + 65);
        }
    }

    console.log(row);
}